code_testing.ts

```diff
@@ -18,13 +18,12 @@ function delay(ms: number)
   return new Promise(resolve => setTimeout(resolve, ms));
 }
 
-function test_company_functions(){
-    let be = new BackendProcessing();
-    let f = be.retrieveCompaniesFromDatabase();
-    f.then((results)=>{
-        console.log(results);
+function test_ibc_retrieval(){
```

Comment:
This was used to test if the function was working properly. (Not part of automated testing.)